### DISCUSSION TITLE
Possible to use a single key for both accepting and selecting completions?

### DISCUSSION THREAD

**Discussion by breed808** [saghen/blink.cmp#6] [Q&A] [answered]
Firstly, thanks for making this plugin! The responsiveness is impressive and the "batteries included" config is rather nice :+1:

Secondly, is it possible to use a single key for multiple actions, such as selecting and accepting completions? Coming from cmp.nvim, I was able to use `<Tab>` for both by using the following config:

```lua
  mapping = {
    ['<Tab>'] = function(fallback)
      if cmp.visible() then
        cmp.select_next_item()
      elseif luasnip.expand_or_jumpable() then
        luasnip.expand_or_jump()
      else
        fallback()
      end
    end,

    ['<S-Tab>'] = function(fallback)
      if cmp.visible() then
        cmp.select_prev_item()
      elseif luasnip.jumpable(-1) then
        luasnip.jump(-1)
      else
        fallback()
      end
    end,
  },
```

It's handy to only need one key (less fingers on less keys) most of the time, but if the above functionality isn't intended for blink.cmp I can unlearn my muscle memory.

**Comment by saghen** [comment, marked as answer]
Yep, what you're looking for should work OOTB on main. If you bind multiple commands to a single key, we'll first try "non-snippet" commands (i.e. select next, accept, etc) and then snippet commands (i.e. jump forward).

The behavior is broken on the latest release but fixed on main, so watch out for a new release

**Reply by OriLiMu** [reply]
how to use tab bind by two commands "select_next" and "snippet_forward" how to make it?
